articles/faq/increase-limit-on-existing-paid-plan.md

```diff
@@ -0,0 +1,5 @@
+---
+ title: "If I stay on my plan, can I increase my daily limits?"
+ description: "Your plan settings will remain as they are now. If you need a larger daily limit, please upgrade to one of our new plans. Please note that daily limits are always based on your account’s sending performance and history with SparkPost and are reviewed on a case by case basis."
+ ---
```

Comment:
add line break between the meta and the actual content

Reply:
done